certified-connectors/Oneflow/apiDefinition.swagger.json

```diff
@@ -1223,117 +1261,101 @@
         "id": {
           "type": "integer",
           "x-ms-summary": "Contract ID",
-          "description":"Contract Id",
+          "description": "Contract Id",
           "format": "int32"
         },
         "lifecycle_settings": {
           "x-ms-visibility": "internal",
-          "x-ms-summary": "Lifecycle Settings",
           "type": "object",
           "properties": {
             "duration": {
               "type": "string",
-              "description": "duration",
-              "x-ms-summary": "Contract Duration"
+              "x-ms-summary": "Contract duration"
             },
             "end_date": {
               "type": "string",
-              "description": "end_date",
-              "x-ms-summary": "Contract End Date"
+              "x-ms-summary": "Contract end date"
             },
             "initial_duration": {
               "type": "string",
-              "description": "initial_duration",
-              "x-ms-summary": "Contract Initial Duration"
+              "x-ms-summary": "Contract initial duration"
             },
             "notice_period": {
               "type": "string",
-              "description": "notice_period",
-              "x-ms-summary": "Contract Notice Period"
+              "x-ms-summary": "Contract notice period"
             },
             "start_date": {
               "type": "string",
-              "description": "start_date",
-              "x-ms-summary": "Contract Start Date"
+              "x-ms-summary": "Contract start date"
             },
             "type": {
               "type": "string",
-              "description": "type",
+              "description": "no_duration/single_period/recurring/recurring_two_periods",
```

Comment:
@DmitryKleshchov please make more detailed descriptions if you want connector users to choose between possible values. You can refer to the [guidelines for creating strings](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings) or [coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#definitions-and-parameters).

Reply:
Hi @v-EgorKozhadei  ! 
Descriptions like above are used only in output models.
 If we make them too long, they won't fit in the output window of the flow constructor, thus becoming harder to understand. As you guessed, these are just the output values of enum. 
The names of parameters are self-explanatory, so we don't need to add more description below, the only option is either to remove description completely, or to have these technical values inside to assist end users in understanding the data that can be contained in these fields: 
![image](https://user-images.githubusercontent.com/15642133/172178827-72602158-bd70-4c9c-a954-5db73d340601.png)

Reply:
@DmitryKleshchov thanks for the answer! I think we can keep it as it is.